modules/rosa-policy-change-management.adoc

```diff
@@ -232,4 +232,8 @@ creation.
 |- Implement change management best practices for customer
 applications and data hosted on the AWS Cloud.
 
-|===
\ No newline at end of file
+|===
+
+^[1]^ For more information on Authentication flow for AWS STS, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/authentication_and_authorization/managing-cloud-provider-credentials#cco-short-term-creds-auth-flow-aws-diagram_cco-short-term-creds[Authentication flow for AWS STS].
+
+^[2]^ For more information on pruning images, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/registry/registry-overview-1#pruning-images_registry-overview[Automatically Pruning Images].
```

Comment:
Actually, I think the footnote style needs to be fixed before merging - it looks like a normal number should be used as in the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables) - this is also the way it's done [elsewhere in the doc](https://92903--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix#rosa-policy-responsibilities_rosa-policy-responsibility-matrix).

Reply:
Fixed the formatting.